src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_data.cljs

```diff
@@ -57,8 +65,25 @@
         [comps/FilterButtons {:buttons [(make-button "Sample" :sample)
                                         (make-button "Aliquot" :aliquot)
                                         (make-button "All" :all)]}]
-        [:div {:style {:padding "2em 0"}} [EntitiesList {:entities (:entities props)
-                                                         :active-filter (:active-filter @state)}]]]))})
 
-(defn render-workspace-data [entities]
-  [WorkspaceData {:entities entities}])
+        [:div {:style {:padding "2em 0"}}
+         (cond
+           (:entities-loaded? @state) [EntitiesList {:entities (:entities @state)
+                                                                               :parent-state state}]
```

Comment:
Maybe there was a bad merge here at some point?  EntitiesList no longer takes a parent-state property, and should take {:active-filter (:active-filter @state)} instead.